### PR TITLE
Fix free pack buying errors

### PR DESCRIPTION
**Pull request by itowhid06**

### CODE REVIEW

File: includes/class-user-subscription.php [hunk 1]
```diff
@@ -462,4 +469,17 @@ public function has_error( $form_settings = null ) {
 
         return false;
     }
+
```
Comment (Sk-Shaikat) [inline includes/class-user-subscription.php]: Add function documentation.

File: includes/class-user-subscription.php [hunk 2]
```diff
@@ -462,4 +469,17 @@ public function has_error( $form_settings = null ) {
...
+    public static function is_free_pack( $pack_id ) {
```
Comment (Sk-Shaikat) [inline includes/class-user-subscription.php]: Add function documentation.

File: includes/class-user-subscription.php (hunk 1 above)
Comment (Sk-Shaikat) [inline includes/class-user-subscription.php]: Add function documentation.

Comment (itowhid06) [reply includes/class-user-subscription.php]: Done

**Pull request merged**